tests/testthat/helper-TealAppDriver.R

```diff
@@ -27,3 +27,34 @@ simple_cdisc_data <- function(datasets = c("ADSL", "ADRS", "ADTTE")) {
   teal.data::join_keys(data) <- teal.data::default_cdisc_join_keys[datasets]
   data
 }
+
+# local app drivers for module testing ----------------------------------------
+# based on examples
+
+app_driver_tm_a_pca <- function() {
+  # Dataset only used once
+  data <- within(teal.data::teal_data(), {
+    require(nestcolor)
+
+    USArrests <- USArrests # nolint: object_name.
+  })
+  teal.data::datanames(data) <- "USArrests"
+
+
+  TealAppDriver$new(
+    data = data,
+    modules = tm_a_pca(
+      dat = teal.transform::data_extract_spec(
+        dataname = "USArrests",
+        select = teal.transform::select_spec(
+          choices = teal.transform::variable_choices(
+            data = data[["USArrests"]],
+            c("Murder", "Assault", "UrbanPop", "Rape")
+          ),
+          selected = c("Murder", "Assault"),
+          multiple = TRUE
+        )
+      )
+    )
+  )
+}
```

Comment:
If this is only going to be used in a single file, why not source it in that file instead of helper?

cc @averissimo @m7pr @kartikeyakirar

Reply:
do you mean creating a separate file for each app_driver ?

Reply:
@kartikeyakirar I think Vedha means keeping it on top of the `test-shinytest2-xxxx.R` file

I try to avoid having anything other than `test_that` calls in the test files, but it makes sense to keep the function definition above the respective tests.

It's not really a helper and this way it avoids having the question "where does it come from?" as @m7pr mentioned in a discussion.

Sounds good to me :+1: